pkg/virt-controller/watch/vm.go

```diff
@@ -965,11 +965,7 @@ func (c *VMController) addStartRequest(vm *virtv1.VirtualMachine) error {
 	return nil
 }
 
-func (c *VMController) startStop(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance) (*virtv1.VirtualMachine, syncError) {
-	runStrategy, err := vm.RunStrategy()
-	if err != nil {
-		return vm, &syncErrorImpl{fmt.Errorf(fetchingRunStrategyErrFmt, err), FailedCreateReason}
-	}
+func (c *VMController) syncRunStrategy(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachineInstance, runStrategy virtv1.VirtualMachineRunStrategy) (*virtv1.VirtualMachine, syncError) {
```

Comment:
The new name doesn't make much sense to me TBH... This function really does start/stop VMs, the name probably should capture that.

Reply:
It actually syncs the VM with the chosen runstrategy, start/stop is a consequence of this. If you really think it should be changed back I'll do, I'm not attached to the new naming.